dotcom-rendering/src/web/components/FrontCard.tsx

```diff
@@ -0,0 +1,50 @@
+import { ArticleDesign } from '@guardian/libs';
+import type { Props as CardProps } from './Card/Card';
+import { Card } from './Card/Card';
+
+type Props = {
+	trail: TrailType;
+} & Partial<CardProps>;
```

Comment:
I wonder. So `trail` is a required prop here because you'd never use `Card` without it. And `trail` itself has some required params which I guess you'd never want to use this component and override. So is the list of override props more of a subset rather than all `Card` props?

E.g.: 

```ts
	linkTo: string;
	format: ArticleFormat;
	headlineText: string;
```

Should never be overridden

```ts
	showQuotes?: boolean;
	byline?: string;
	showByline?: boolean;
	webPublicationDate?: string;
	imageUrl?: string;
	imagePosition?: ImagePositionType;
	imagePositionOnMobile?: ImagePositionType;
	imageSize?: ImageSizeType;
	trailText?: string;
	avatar?: AvatarType;
	showClock?: boolean;
	mediaType?: MediaType;
	mediaDuration?: number;
	kickerText?: string;
	showPulsingDot?: boolean;
	showSlash?: boolean;
	starRating?: number;
	minWidthInPixels?: number;
	dataLinkName?: string;
	branding?: Branding;
	supportingContent?: DCRSupportingContent[];
	snapData?: DCRSnapType;
	containerPalette?: DCRContainerPalette;
	containerType?: DCRContainerType;
	showAge?: boolean;
	discussionId?: string;
	isDynamo?: true;
```

_Could_. But maybe only some of these should_ be overridable? Like:

```ts
	showQuotes?: boolean;
	showByline?: boolean;
	imageUrl?: string;
	imagePosition?: ImagePositionType;
	imagePositionOnMobile?: ImagePositionType;
	imageSize?: ImageSizeType;
	trailText?: string;
	avatar?: AvatarType;
	kickerText?: string;
	showPulsingDot?: boolean;
	showSlash?: boolean;
	supportingContent?: DCRSupportingContent[];
	snapData?: DCRSnapType;
	containerPalette?: DCRContainerPalette;
	containerType?: DCRContainerType;
	showAge?: boolean;
	discussionId?: string;
	isDynamo?: true;
```

What do you think? Maybe we can just have these props set explicitly here and default the rest to whatever is passed in on the `trail` object?

In the future, some of these can be removed, like `showPulsingDot` or `showQuotes`, which can be worked out inside the Card using `format`

Reply:
I'm open to the option of being a bit more restrictive with what props we allow overriding but I'm curious to what the benefit of this would be and if its worth the extra complexity of doing this?

Reply:
Going to merge this down for now and raise a subsequent PR to restrict the props!